_posts/2022-04-27-cockpit-268.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cockpit 268
+author: garrett
+date: '2022-04-27'
+tags: cockpit
+slug: cockpit-268
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 268:
+
+
+## Software Updates: Only install kpatches
+
+Live patch a kernel without having to reboot or restart services with the new "Install kpatch updates" button. This is especially useful to fix an urgent kernel vulnerability with no downtime.
```

Comment:
"especially useful to fix an urgent kernel vuln" makes it sound like there's a specific active one we should be worried about which this Cockpit release might somehow help with.  Maybe a part of that is that the audience reading this knows already why they'd want to use this feature in the general case.

Maybe we should try and phrase this a bit more vaguely like "This allows patching of urgent kernel vulnerabilities without rebooting." or so.

Reply:
Thanks! I've updated the text in the PR and rebuilt the release notes; it should be in the preview in a few moments.